spyder/plugins/workingdirectory/confpage.py

```diff
@@ -62,32 +66,34 @@ def setup_page(self):
         startup_group.setLayout(startup_layout)
 
         # Console Directory
-
-        console_group = QGroupBox(_("Console directory"))
-        console_label = QLabel(_("The working directory for new consoles is:"))
+        console_group = QGroupBox(_("New consoles"))
+        console_label = QLabel(
+            _("The working directory for new IPython consoles is:")
+        )
         console_label.setWordWrap(True)
         console_bg = QButtonGroup(console_group)
         console_project_radio = self.create_radiobutton(
-                                _("The current project directory "
-                                  "or user home directory "
-                                  "(if no project is active)"),
-                                'console/use_project_or_home_directory',
-                                True,
-                                button_group=console_bg)
+            _("The project or user home directory"),
+            'console/use_project_or_home_directory',
+            button_group=console_bg
+        )
         console_cwd_radio = self.create_radiobutton(
-                                _("The current working directory"),
-                                'console/use_cwd',
-                                False,
-                                button_group=console_bg)
-
+            _("The current working directory"),

```

Comment:
```suggestion
            _("The working directory of the current console"),
```

It seems a bit confusing to say that the "current working directory" will be the "current working directory", so this clarifies where that comes from. A tooltip could also potentially be added, if needed.